.github/renovate.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": [
+    "config:base"
```

Comment:
Please add ":disableDependencyDashboard" - to disable the dependency Dashboard which creates a new issue always when the PR merge is not successful. Which is again double task for us to check and close both issue and PR.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/767/commits/0600f0de04cd907f0e18604d3b4d4bdcc69bb185

Reply:
It is better to use ":disableDependencyDashboard" . check here https://github.com/renovatebot/renovate/discussions/11330#discussioncomment-1210241

You can do it like below,

**"extends": ["config:base", ":disableDependencyDashboard"],**

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/767/commits/208ce5a679059b20a367f5a1fdcaf74f907b1844